packages/cli/src/databases/migrations/mysqldb/1675940580449-PurgeInvalidWorkflowConnections.ts

```diff
@@ -0,0 +1,84 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { getTablePrefix, logMigrationEnd, logMigrationStart } from '@db/utils/migrationHelpers';
+import { NodeTypes } from '@/NodeTypes';
+import { IConnections, INode } from 'n8n-workflow';
+
+export class PurgeInvalidWorkflowConnections1675940580449 implements MigrationInterface {
+	name = 'PurgeInvalidWorkflowConnections1675940580449';
+
+	async up(queryRunner: QueryRunner): Promise<void> {
+		logMigrationStart(this.name);
+
+		const tablePrefix = getTablePrefix();
+
+		const workflows: Array<{ id: number; nodes: string; connections: string }> =
+			await queryRunner.query(`
+			SELECT id, nodes, connections
+			FROM \`${tablePrefix}workflow_entity\`
+		`);
+
+		const nodeTypes = NodeTypes();
+
+		workflows.forEach(async (workflow) => {
+			let connections: IConnections = JSON.parse(workflow.connections);
+			const nodes: INode[] = JSON.parse(workflow.nodes);
```

Comment:
Just like postgres, we get those variables as objects already, so we shouldn't `JSON.parse` them.